solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -574,306 +584,124 @@ describe("audius-data", function () {
-      message,
-      ethAccount,
-      baseAuthorityAccount,
-      handleBytesArray,
-      bumpSeed,
-      metadata,
-      newUserKeypair,
-      userId,
-      userStorageAccount: newUserAcctPDA,
-      adminStoragePublicKey: adminStorageKeypair.publicKey,
       ...getURSMParams(),
     });
 
-    // Init AuthorityDelegationStatus for a new authority
-    const userAuthorityDelegateKeypair = anchor.web3.Keypair.generate();
-    const authorityDelegationStatusSeeds = [
-      Buffer.from("authority-delegation-status", "utf8"),
-      userAuthorityDelegateKeypair.publicKey.toBytes().slice(0, 32),
-    ];
-
-    const authorityDelegationStatusRes = await PublicKey.findProgramAddress(
-      authorityDelegationStatusSeeds,
-      program.programId
-    );
-    const authorityDelegationStatusPda = authorityDelegationStatusRes[0];
-
-    const initAuthorityDelegationStatusArgs = {
-      accounts: {
-        delegateAuthority: userAuthorityDelegateKeypair.publicKey,
-        authorityDelegationStatusPda: authorityDelegationStatusPda,
-        payer: provider.wallet.publicKey,
-        systemProgram: SystemProgram.programId,
-      },
-      signers: [userAuthorityDelegateKeypair],
-    };
-
-    await program.rpc.initAuthorityDelegationStatus(
-      "authority_name",
-      initAuthorityDelegationStatusArgs
-    );
-
-    // New sol key that will be used as user authority delegate
-    const userAuthorityDelegateSeeds = [
-      newUserAcctPDA.toBytes().slice(0, 32),
-      userAuthorityDelegateKeypair.publicKey.toBytes().slice(0, 32),
-    ];
-    const res = await PublicKey.findProgramAddress(
-      userAuthorityDelegateSeeds,
-      program.programId
-    );
-    const userAuthorityDelegatePDA = res[0];
-    const userAuthorityDelegateBump = res[1];
-
-    const addUserAuthorityDelegateArgs = {
-      accounts: {
-        admin: adminStorageKeypair.publicKey,
-        user: newUserAcctPDA,
-        userAuthorityDelegatePda: userAuthorityDelegatePDA,
-        userAuthority: newUserKeypair.publicKey,
-        payer: provider.wallet.publicKey,
-        systemProgram: SystemProgram.programId,
-      },
-      signers: [newUserKeypair],
-    };
-
-    await program.rpc.addUserAuthorityDelegate(
-      baseAuthorityAccount,
-      handleBytesArray,
-      bumpSeed,
-      userAuthorityDelegateKeypair.publicKey,
-      addUserAuthorityDelegateArgs
-    );
-
     const acctState = await program.account.userAuthorityDelegate.fetch(
-      userAuthorityDelegatePDA
+      delegate.userAuthorityDelegatePDA
     );
     const userStoragePdaFromChain = acctState.userStorageAccount;
     const delegateAuthorityFromChain = acctState.delegateAuthority;
     expect(userStoragePdaFromChain.toString(), "user storage pda").to.equal(
-      newUserAcctPDA.toString()
+      delegate.userAccountPDA.toString()
     );
     expect(
-      userAuthorityDelegateKeypair.publicKey.toString(),
+      delegate.userAuthorityDelegateKeypair.publicKey.toString(),
```

Comment:
can we call the returned result something other than `delegate`? The params are hard to understand here.